index.rst

```diff
@@ -505,15 +505,117 @@ on the producer end.
+| 7 days                      |  64                            |  64 x 164 GB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 7 days                      |  128                           |  128 x 82 GB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 14 days                     |  1                             |  1 x 21 TB                     |
++-----------------------------+--------------------------------+--------------------------------+
+| 14 days                     |  16                            |  16 x 1.3 TB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 14 days                     |  32                            |  32 x 656 GB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 14 days                     |  64                            |  64 x 328 GB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 14 days                     |  128                           |  128 x 164 GB                  |
++-----------------------------+--------------------------------+--------------------------------+
+| 30 days                     |  1                             |  1 x 45 TB                     |
++-----------------------------+--------------------------------+--------------------------------+
+| 30 days                     |  16                            |  16 x 2.8 TB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 30 days                     |  32                            |  32 x 1.4 TB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 30 days                     |  64                            |  64 x 703 GB                   |
++-----------------------------+--------------------------------+--------------------------------+
+| 30 days                     |  128                           |  128 x 352 GB                  |
++-----------------------------+--------------------------------+--------------------------------+
+
+In the experiments here, an increased number of partitions decreases the total time to distribute alerts on
+both the producer and consumer side and allows consumers to use parallel readers.  A number of partitions
+greater than one is recommended, and if the compute pipeline is parallelized by CCD, one partition per CCD,
+would be a reasonable choice.  More partitions can lead to higher throughput but requires increased memory.
+As a rule of thumb, Confluent recommends no more than 100 * b * r partitions per broker, where b is the number
```

Comment:
Here (and throughout the technote) there's an unfortunate terminology clash between “broker” in the sense of the LSST mini broker, and “broker” in the sense of Kafka nodes. I don't think you should try to rewrite to avoid that issue, but maybe consider just dropping in a note somewhere to make it clear?